demisto_sdk/tests/integration_tests/content_create_artifacts_integration_test.py

```diff
@@ -62,3 +67,57 @@ def test_duplicate_file_failure(mock_git):
         result = runner.invoke(main, [ARTIFACTS_CMD, '-a', temp, '--no-zip'])
 
     assert result.exit_code == 1
+
+
+def test_test_specific_pack_creation(repo):
+    """Test the -p flag for specific packs creation
+    """
+    pack_1 = repo.setup_one_pack('Pack1')
+    pack_1.pack_metadata.write_json(
+        {
+            'name': 'Pack Number 1',
+        }
+    )
+
+    pack_2 = repo.setup_one_pack('Pack2')
+    pack_2.pack_metadata.write_json(
+        {
+            'name': 'Pack Number 2',
+        }
+    )
+
+    with ChangeCWD(repo.path):
+        with temp_dir() as temp:
+            runner = CliRunner(mix_stderr=False)
+            result = runner.invoke(main, [ARTIFACTS_CMD, '-a', temp, '-p', 'Pack1'])
+
+    assert result.exit_code == 0
+    assert 'Pack1' in result.stderr
+    assert 'Pack2' not in result.stderr
```

Comment:
if you expect a successful run (error code is 0), why do you expect anything in the stderr?